tests/unit/rest.py

```diff
@@ -556,3 +563,21 @@ def increment():
 
         self.client.post('/increment')
         assert self.num_calls == 1
+
+
+def test_dont_log(client, caplog):
+    @rest.rest_call('POST', '/some-path', Schema({
+        'public': basestring,
+        'private': basestring,
+        'stuff': basestring,
+    }), dont_log=('private',))
+    def some_call(public, private, stuff):
+        pass
+
+    with caplog.atLevel(logging.DEBUG):
```

Comment:
Was looking up what this is and it looks like `atlevel` is [deprecated](https://github.com/eisensheng/pytest-catchlog/blob/84de498d7a80378e36645cd2907226efc24cac65/CHANGES.rst). Seems like it's the same functionality, but method names changed to follow pep8